src/ape/managers/converters.py

```diff
@@ -91,7 +92,67 @@ def convert(self, value: str) -> AddressType:
         return to_checksum_address(value)
 
 
-hex_address_converter = HexAddressConverter(None, None)  # type: ignore
+hex_address_converter = HexAddressConverter(None, None, None)  # type: ignore
+
+
+class ListTupleConverter(ConverterAPI):
+    """
+    A converter that converts any tuple or list of items
```

Comment:
```suggestion
    A converter that converts any tuple or list of items to a list of
```

Reply:
Hmm, this statement is just way off entirely, isn't it?